tests/dummy/config/environment.js

```diff
@@ -21,6 +21,9 @@ module.exports = function(environment) {
       // Here you can pass flags/options to your application instance
       // when it is created
     },
+    fastboot: {
+      hostWhitelist: [/^localhost:\d+$/]
```

Comment:
required to `ember s` locally, ember fastboot asserts on the hostname that is serving the app